src/components/share-drawer/messages.ts

```diff
@@ -0,0 +1,25 @@
+import { ShareType } from 'audius-client/src/common/store/ui/share-modal/types'
+
+const shareTypeMap: Record<ShareType, string> = {
+  track: 'Track',
+  profile: 'Profile',
+  album: 'Album',
+  playlist: 'Playlist'
+}
+
+export const messages = {
```

Comment:
Just thinking ahead to when we get around to actually using something like [l18n](https://react.i18next.com/), we'll probably need these to not be functions but plain strings which are then interpolated after the fact. I'm not sure we need it to work that way right now, since I don't know which internationalization tool we're going to use and when, but worth considering when making these `messages` objects.

Maybe we roll our own interpolator in the meantime? Might be something to add to prod eng sync

Reply:
yeah totally agreed, would be open to hear what folks think at a prod sync